### ISSUE TITLE
Korean: leftover non-name test failures

### ISSUE THREAD

**Issue by scossu** [lcnetdev/scriptshifter#42]
@hyoungl Attached here is a list of the last failing Korean tests from the test strings you provided. I thought it would be more practical for you to review the log as a whole and comment on the individual issues, as many of them seem related to personal names. 
[korean_tests.log](https://github.com/lcnetdev/scriptshifter/files/12250858/korean_tests.log)

Thanks.

**Comment by hyoungl** [comment]
Ok. I'll start looking into the list.

**Comment by scossu** [comment]
Updated list:  [korean_nonames_tests.log](https://github.com/lcnetdev/scriptshifter/files/12508372/korean_nonames_tests.log)

I noticed this: 

```
- Am yebang: 5-kaji saekkal ŭl mŏgŏra
+ Am yebang: 5-kaji saekkkal ŭl mŏgŏra
?                        +
 : S2R transliteration error for korean_nonames!
Original: 암 예방 : 5가지 색깔 을 먹어라
```

This is a result of  the "kkk -> kk" replacement rule. Is "kkk" legit here?

**Comment by hyoungl** [comment]
No. "kkk" is never allowed in the current rules. It always has to be turned into "kk"

**Comment by scossu** [comment]
[test_korean.log](https://github.com/lcnetdev/scriptshifter/files/13262333/test_korean.log)

New test log.

**Comment by scossu** [comment]
I have found a few names that had split digraph, which I have corrected. 

All remaining ones (139) look like sentences with proper names which won't transliterate correctly anyway, or possibly legitimate errors on names. 

[test_korean.log](https://github.com/lcnetdev/scriptshifter/files/13926778/test_korean.log)

**Comment by scossu** [comment]
More fixes in #91. Updated list: 

[test_korean-20240113.log](https://github.com/lcnetdev/scriptshifter/files/13930359/test_korean-20240113.log)

**Comment by scossu** [comment]
All tests pass now. Closing.
